src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/HttpVersionCompletionsAttribute.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Net;
+using System.Reflection;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// A completer for HTTP version names.
+    /// </summary>
+    internal sealed class HttpVersionCompletionsAttribute : ArgumentCompletionsAttribute
+    {
+        public static readonly string[] AllowedVersions;
+
+        static HttpVersionCompletionsAttribute()
+        {
+            FieldInfo[] fields = typeof(HttpVersion).GetFields(System.Reflection.BindingFlags.Static | System.Reflection.BindingFlags.Public);
+
+            if (fields.Length == 0)
+            {
+                AllowedVersions = Array.Empty<string>();
+                return;
+            }
```

Comment:
This seems unneeded, as it will be breaking change if `HttpVersion` removes those fields in future.